pkg/controller/volume/persistentvolume/util/util.go

```diff
@@ -64,6 +64,13 @@ const (
 	// recognize dynamically provisioned PVs in its decisions).
 	AnnDynamicallyProvisioned = "pv.kubernetes.io/provisioned-by"
 
+	// AnnMigratedTo annotation is added to a PVC and PV that is supposed to be
+	// dynamically provisioned/deleted by by its corresponding CSI driver
+	// through the CSIMigration feature flags. When this annotation is set the
+	// Kubernetes components will "stand-down" and the external-provisioner will
+	// act on the objects
+	AnnMigratedTo = "volume.beta.kubernetes.io/migrated-to"
```

Comment:
Sorry I didn't catch this earlier, but putting "alpha/beta/ga" into annotation keys are no longer the best practice. It made graduating from beta->ga more complex because you need to handle both, and potentially backfill the ga annotation on existing objects. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions

Reply:
no worries, we'll fix it in master and bring it into the cherry-pick

Reply:
updated in cp and PR in master: https://github.com/kubernetes/kubernetes/pull/88032